src/dmd/globals.d

```diff
@@ -126,6 +126,7 @@ extern (C++) struct Param
     bool showColumns;       // print character (column) numbers in diagnostics
     bool vtls;              // identify thread local variables
     bool vtemplates;        // collect and list statistics on template instantiations
+    bool vtemplatesListInstances; // collect and list statistics on template instantiations origins
```

Comment:
Note for the future: This should probably become an `enum` if we add more modes

Reply:
Added TODO in comment.